src/domain/accounts/entities/upsert-account-data-settings.dto.entity.ts

```diff
@@ -0,0 +1,23 @@
+import { z } from 'zod';
+
+export class UpsertAccountDataSettingsDto
+  implements z.infer<typeof UpsertAccountDataSettingsDtoSchema>
+{
+  accountDataSettings: {
+    dataTypeName: string;
+    enabled: boolean;
+  }[];
+
+  constructor(props: UpsertAccountDataSettingsDto) {
+    this.accountDataSettings = props.accountDataSettings;
+  }
+}
+
+export const UpsertAccountDataSettingDtoSchema = z.object({
+  dataTypeName: z.string(),
```

Comment:
As before regarding prefix.

Reply:
Changed in [2a6e988](https://github.com/safe-global/safe-client-gateway/pull/1715/commits/2a6e988cbd0a6970b9aff2d1efd9b28d43342955)